pkg/cli/create_helm.go

```diff
@@ -380,7 +380,7 @@ func (cmd *createHelm) addVCluster(ctx context.Context, vClusterConfig *config.C
 		return nil
 	}
 
-	err = platform.ApplyPlatformSecret(ctx, cmd.LoadedConfig(cmd.log), cmd.kubeClient, "", cmd.Namespace, cmd.Project, "", "", false)
+	err = platform.ApplyPlatformSecret(ctx, cmd.LoadedConfig(cmd.log), cmd.kubeClient, "", cmd.Namespace, cmd.Project, "", "", false, cmd.LoadedConfig(cmd.log).Platform.CertificateAuthorityData)
```

Comment:
Shouldn't this come from the flag? Why is this read from the config, while the other options are read from the flag?

Reply:
I'm not sure what's the preferred way to do this here.
I can change it for a flag, which will mean that `vcluster create vcluster` with driver `helm` will get a new flag. Do you think this should be a desired behavior?